src/main/java/org/sciborgs1155/robot/flywheel/RealFlywheel.java

```diff
@@ -0,0 +1,57 @@
+package org.sciborgs1155.robot.flywheel;
+
+import static edu.wpi.first.units.Units.*;
+import static org.sciborgs1155.robot.Ports.Shooter.Flywheel.*;
+import static org.sciborgs1155.robot.flywheel.FlywheelConstants.*;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkFlex;
+import com.revrobotics.RelativeEncoder;
+import java.util.Set;
+import org.sciborgs1155.lib.SparkUtils;
+import org.sciborgs1155.lib.SparkUtils.Data;
+import org.sciborgs1155.lib.SparkUtils.Sensor;
+
+public class RealFlywheel implements FlywheelIO {
+  private final CANSparkFlex topMotor;
+  private final CANSparkFlex bottomMotor;
+  private final RelativeEncoder encoder;
+
+  public RealFlywheel() {
+
+    topMotor = SparkUtils.createSparkFlex(TOP_MOTOR, false, IdleMode.kBrake, CURRENT_LIMIT);
+    bottomMotor = SparkUtils.createSparkFlex(BOTTOM_MOTOR, false, IdleMode.kBrake, CURRENT_LIMIT);
+    encoder = topMotor.getEncoder();
+
+    encoder.setPositionConversionFactor(POSITION_FACTOR.in(Radians));
+    encoder.setVelocityConversionFactor(VELOCITY_FACTOR.in(RadiansPerSecond));
+
```

Comment:
configure the encoder velocity filter with `setMeasurementPeriod` and `setAverageDepth`
the values 6328 uses for their swerve (probably reasonable) are a measurement period of 10 and an average depth of 2